src/main/kubectl.ts

```diff
@@ -315,8 +316,14 @@ export class Kubectl {
     const fileWriteStream = fs.createWriteStream(this.path, { mode: 0o755 });
     const pipeline = promisify(stream.pipeline);
 
-    await pipeline(downloadStream, fileWriteStream);
-    logger.debug("kubectl binary download finished");
+    try {
+      await pipeline(downloadStream, fileWriteStream);
+      await fs.promises.chmod(this.path, 0o755);
```

Comment:
Do you still need this given that the write stream is now created with that mode? I recall it wasn't created with that mode previously, maybe there was a reason (i.e. don't make it executable until it is successfully downloaded)?

Reply:
Maybe, but I was mostly just reverting the bug from before.